.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,107 @@
+name: Bug Report
+description: Report a bug encountered while operating Kubernetes
```

Comment:
If this template will be the same for all the distro repos (the distro itself and the modules) maybe we should talk about operating kubernetes? we could just use "the software" maybe?